Resources/Locale/en-US/power/components/charger.ftl

```diff
@@ -1 +1,2 @@
-charger-examine = Charges [color={$color}]{$chargeRate}W[/color] per second.
\ No newline at end of file
+charger-examine = Charges [color={$color}]{$chargeRate}W[/color] per second.
```

Comment:
change W to J or remove per second we are big boys and know what watts are :trollface:

Reply:
what... why? It charges watts per second, isn't it?

Reply:
watts is joules per second bruh